pkgs/servers/monitoring/mackerel-agent/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv, lib, buildGoModule, fetchFromGitHub, makeWrapper, iproute, nettools }:
+
+buildGoModule rec {
+  pname = "mackerel-agent";
+  version = "0.68.2";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "mackerelio";
+    repo = pname;
```

usually ordering, nit
```suggestion
    owner = "mackerelio";
    repo = pname;
    rev = "v${version}";
```